benchmarks/microbenchmarks/operator_inp_logs/hf_train/AlbertForMaskedLM_training.txt

```diff
@@ -0,0 +1,115 @@
+Operator: aten._log_softmax.default
+cnt: 1, '((T([2048, 30000], th.float16), 1, False), {})'
+Operator: aten._log_softmax_backward_data.default
+cnt: 1, '((T([2048, 30000], th.float16), T([2048, 30000], th.float16), 1, th.float16), {})'
+Operator: aten._softmax.default
+cnt: 12, '((T([4, 64, 512, 512], th.float16), -1, False), {})'
+Operator: aten._softmax_backward_data.default
+cnt: 12, '((T([4, 64, 512, 512], th.float16), T([4, 64, 512, 512], th.float16), -1, th.float16), {})'
+Operator: aten._to_copy.default
+cnt: 1, "((T([4, 1, 1, 512], th.float32),), {'dtype': th.float16})"
+Operator: aten._unsafe_view.default
+cnt: 36, '((T([4, 64, 512, 64], th.float16), [256, 512, 64]), {})'
```

Comment:
and then when there are strides, I'd suggest we spend the characters calling it out explicitly: `stride=[256, 512, 64]`

Reply:
Done